### DISCUSSION TITLE
Can I find what module or view rendered each part of html ?

### DISCUSSION THREAD

**Discussion by DrewBrasher** [OrchardCMS/OrchardCore#11568] [Q&A] [answered]
I think I remember seeing somewhere that there is a feature that can be enabled that will add comments to the html to show information about what rendered different sections of the html but I can't find anything about it now. Is there a feature like that?

**Comment by Skrypt** [comment, marked as answer]
Mini Profiler 🐰 

And you can also use console log.

https://docs.orchardcore.net/en/dev/docs/reference/modules/Contents/#razor-console-log
https://docs.orchardcore.net/en/dev/docs/reference/modules/Contents/#logging-to-the-browser-console
